src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -340,4 +364,23 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 		}
 		return undefined;
 	}
+
+	public fileToBase64 = (filepath, getType) => {
+		return new Promise<string | ArrayBuffer>(async resolve => {
+			let response = await fetch(filepath);
+			let blob = await response.blob();
+			if (getType) {
```

Comment:
What is this doing? It seems like this function can return two different types of results depending on whether getType was true or not?